uint8 robot
---
bool success